homeassistant/components/renson/sensor.py

```diff
@@ -292,6 +281,10 @@ def _handle_coordinator_update(self) -> None:
 
         if self.raw_format:
             self._attr_native_value = value
+        elif self.entity_description.device_class == SensorDeviceClass.ENUM:
+            self._attr_native_value = slugify(
```

Comment:
This is breaking, we cannot avoid that, but we need to document it and mark the PR as such.

We should not slugify for values. Instead a mapping needs to be made and looked up.